sdk/go/common/resource/resource_state.go

```diff
@@ -40,18 +40,26 @@ type State struct {
 	PropertyDependencies    map[PropertyKey][]URN // the set of dependencies that affect each property.
 	PendingReplacement      bool                  // true if this resource was deleted and is awaiting replacement.
 	AdditionalSecretOutputs []PropertyKey         // an additional set of outputs that should be treated as secrets.
-	Aliases                 []URN                 // TODO
+	Aliases                 []Alias               // TODO
```

Comment:
We don't want to change state, it will cause pain if people up/downgrade their pulumi version for the same stack.
Just leave this as is (it's only ever one URN at most)